test/DebugInfo/DebugInfoLLVMArg.ll

```diff
@@ -3,13 +3,24 @@
 ; RUN: llvm-as %s -o %t.bc
 ; RUN: llvm-spirv %t.bc -o %t.spv --spirv-allow-extra-diexpressions
 ; RUN: llvm-spirv %t.spv -to-text -o %t.spt
-; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
+; RUN: FileCheck < %t.spt %s --check-prefixes=CHECK-SPIRV,CHECK-SPIRV-OCL
 ; RUN: llvm-spirv -r -emit-opaque-pointers %t.spv -o %t.rev.bc
 ; RUN: llvm-dis < %t.rev.bc | FileCheck %s --check-prefix=CHECK-LLVM
 
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -o %t.spv --spirv-debug-info-version=nonsemantic-shader-200
```

Comment:
Not sure if we set up the proper BKM for that, but just curious - isn't it better to copy the test inside `NonSemantic/Shader200` folder and do these checks there?

Reply:
I see it this way:
Generic tests should preserve in test/DebugInfo (. and/or Generic and/or X86)
NonSemantic-specific issues should be in NonSemantic (. and or ./(version) directory)
OpenCL - specific should remain in OpenCL dir.

The test is somewhat generic (it tests both W/A for OpenCL DebugInfo and NonSemantic info)

Reply:
Ok, I actually don't mind. We'll refactor layout if we find it more useful one day.